spec/controllers/api/v1/companies_controller_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::CompaniesController, type: :controller do
+
+  describe "GET #show" do
+    let(:response_body) { JSON.parse(response.body) }
+    let(:company) { create(:company) }
+
+    it "is successful" do
+      get :show, id: company.id, format: :json
+
+      expect(:success)
+    end
+
+    it 'returns company with correct attributes' do
+      get :show, id: company.id, format: :json
+      json_company = response_body['company']
+
+      expect(json_company['name']).to be_instance_of(String)
+      expect(json_company['jobs']).to be_instance_of(Array)
+    end
```

Comment:
It's great to test that you are getting the right class of data back, but it might be worth testing something a little more specific, like giving the company a name and testing that you get that actual name back.

Since you've got a serializer in place this would be a way to make sure that you are getting back the data properly if later you wanted to do any formatting in that serializer.